google-maps/package.json

```diff
@@ -62,6 +62,8 @@
     "@capacitor/ios": "^5.0.0",
     "@ionic/eslint-config": "^0.3.0",
     "@ionic/prettier-config": "^1.0.1",
+    "@ionic/swiftlint-config": "^1.1.2",
```

Comment:
This is not needed as the swift lint config was changed to use just references from the `swiftlint.config.js` file.

Reply:
Did something go wrong with the last merge commit @theproducer? I didnt touched this line.

Reply:
Possibly.  I'll remove it.